### ISSUE TITLE
Resolve "ERR_UNSUPPORTED_ESM_URL_SCHEME" Error in Windows

### ISSUE THREAD

**Issue by Railly** [vadimdemedes/pastel#55]
Hi there
First off, thank you for your amazing work, I really enjoy how you approach and adjust the CLI paradigm like Next.js.

Now I've encountered an issue on Windows where Node.js's ESM loader throws an `ERR_UNSUPPORTED_ESM_URL_SCHEME` error when trying to dynamically import a module using an absolute path.

This line of code in `read-commands.js` is causing the issue:
```javascript
const m = (await import(filePath));
```

By using `pathToFileUrl` from `node:url` worked for me, this should ensure compatibility with Windows without affecting other OS.
```javascript
import { pathToFileURL } from 'node:url';
...
const filePath = path.join(directory, file);
const fileURL = pathToFileURL(filePath);
const m = (await import(fileURL));
```

Note: Could you please confirm if you will be working on that? If not, I can contribute effortlessly.

**Comment by vadimdemedes** [comment]
Thank you, I'd appreciate a PR ;)

**Comment by AugusDogus** [comment]
Just so you're aware @vadimdemedes, it appears that the version [uploaded to the npm registry](https://registry.npmjs.org/pastel/-/pastel-2.0.1.tgz) does not have this fix, although the version on Github releases does have it.

**Comment by vadimdemedes** [comment]
@AugusDogus Damn, I forgot the `prepare` script, so code wasn't recompiled when publishing. Thanks for reporting, pushing a patch right now.

**Comment by vadimdemedes** [comment]
@AugusDogus Published a 2.0.2.
